api/src/main/resources/workbench.yaml

```diff
@@ -1141,6 +1141,22 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
+  /v1/cron/exportToRDR:
+    get:
+      security: []
+      tags:
+        - offlineRDRExport
```

Comment:
Side note: don't forget to rename "RDR" --> "Rdr" for proper camel-casing in these YAML files as well. Sorry for the busywork, but experience shows it's super helpful to have consistency in how we handle naming issues like this.